### PR TITLE
Feature bgdidic 645 epics

### PR DESCRIPTION
**Pull request by ltclm**
[Testlink](https://mf-chsdi3.dev.bgdi.ch/feature_BGDIDIC-645_epics/shorten/8f47a0e4a0)
This will implement the new data catalogue of e-pics provided by ETHZ.
These images will be added to the layer: ch.swisstopo.lubis-luftbilder-dritte-firmen

### CODE REVIEW

**Comment by davidoesch** [comment]
Der DOI Link ist das Tüpfelchen auf dem i .
Thx 🙏

File: chsdi/templates/htmlpopup/lubis.mako [hunk 1]
```diff
@@ -217,7 +211,17 @@ shop_url = request.registry.settings['shop_url']
...
-    <tr><th class="cell-left">${_('tt_lubis_bildpfad')}</th>         <td>${filename or '-'}</td></tr>
+    <tr><th class="cell-left">${_('tt_lubis_bildpfad')}</th>
+
+% if filename and filename.startswith('http'):
+        <td><a href="${filename or '-'}" target="_blank">${filename or '-'}</a></td></tr>
+% else:
+        <td>${filename or '-'}</td></tr>
+% endif
+
+% if 'doi_link' in c['attributes'] and c['attributes']['doi_link'] and c['attributes']['doi_link'].startswith('http'):
```
Comment (b4l) [inline chsdi/templates/htmlpopup/lubis.mako:222]: this could be written as: `if c['attributes'].get('doi_link', '').startswith('http'):`

Comment (ltclm) [reply chsdi/templates/htmlpopup/lubis.mako]: thanks good point!

**Review by b4l — APPROVED** [review]
ETHZ file links have no extension, otherwise LGTM

**Pull request merged**